core/infra/service/flatbuffer_service.cpp

```diff
@@ -26,6 +29,165 @@
 namespace flatbuffer_service {
 
 /**
+ * CreateCommandDirect
+ */
+flatbuffers::Offset<void> CreateCommandDirect(
```

Comment:
Is it better made each `CreateXXXCommanndDirect()` ?
Becouse, obj is not only an element.